### DISCUSSION TITLE
Having trouble with config files

### DISCUSSION THREAD

**Discussion by timeades** [webdriverio/webdriverio#6609] [Q&A]
So I'm trying to create various config files so I can run my test, in browser, headless and mobile web using devtools and browser.emulateDevice. I have my headless config running fine but I can't for the life of me get the browser.emulateDevice running and was wondering if some extra pairs of eyes might be able to point out where I'm going wrong. Cheers

// wdio.ios.config.js

const wdioConfig = require('../wdio.conf.js');

wdioConfig.config.debug = false;

exports.config =  {
    capabilities: [
        // browser defined here
        {
            browser: 'chrome',
        },
        // define device and performance
            browser.emulateDevice('iPhone X'),
            browser.enablePerformanceAudits({
            networkThrottling: 'Good 4G',
            cpuThrottling: 4,
            formFactor: 'mobile'
    }),
        // ...
    ],
    services: ['devtools']
}

-----------------------------------------------------------------
UPDATE

OK I have continued to play and now my config looks like this // wdio.ios.config.js

const wdioConfig = require('../wdio.conf.js');

wdioConfig.config.debug = false;

wdioConfig.config.capabilities = [
    {
        browserName: 'chrome',
        'goog:chromeOptions': {
            args: ['browser.emulateDevice(\'iPhone X\')', 'disable-web-security'],
        },
    },
];
exports.config = wdioConfig.config;

and when I run npx wdio run xxxxxxxxxxx I see the the args are supposedly being passed but I don't get an emulated view just a normal browser 
 capabilities: {
    alwaysMatch: { browserName: 'chrome', 'goog:chromeOptions': [Object] },
    firstMatch: [ {} ]
  },
  desiredCapabilities: { browserName: 'chrome', 'goog:chromeOptions': { args: [Array] } }
}

**Comment by christian-bromann** [comment]
Can you provide the error message or explain in a bit more detail what you are struggling with?
